ols/app/endpoints/ols.py

```diff
@@ -115,6 +113,18 @@ def conversation_request(llm_request: LLMRequest) -> LLMResponse:
     return llm_response
 
 
+def retrieve_conversation_id(llm_request: LLMRequest) -> str:
```

Comment:
Should be defined at the top, but you'll be moving it around I guess.

Reply:
yes